src/main/resources/application.properties

```diff
@@ -6,10 +6,13 @@ spring.cache.cache-names=events
 spring.cache.caffeine.spec=expireAfterWrite=120s
 
 server.port=8443
-server.ssl.key-store=classpath:keystore.jks
-server.ssl.key-store-password=password
-server.ssl.key-password=secret
+#server.ssl.key-store=classpath:keystore.jks
```

Comment:
@jankeromnes instead of disabling it here. could you try to add a `override4gitpod.properties` file and try to unset ssl related things? the command can be then changed to something like this:

`java -jar target/issues-dashboard-0.0.1-SNAPSHOT.jar --spring.config.location=classpath:/application.properties,classpath:/override4gitpod.properties`

cf. https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-external-config.html